<!DOCTYPE html>
<html lang='en'>
<head>
<title>wxwidgets-3.2.patch « amule « testing - aports - Alpine packages build scripts
</title>
<meta name='generator' content='cgit v1.2.3'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='shortcut icon' href='//alpinelinux.org/alpine-logo.ico'/>
<link rel='alternate' title='Atom feed' href='http://git.alpinelinux.org/aports/atom/testing/amule/wxwidgets-3.2.patch?h=master' type='application/atom+xml'/>
<link rel='vcs-git' href='https://git.alpinelinux.org/aports' title='aports Git repository'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/'><img src='//wiki.alpinelinux.org/images/alogo.svg' alt='cgit logo'/></a></td>
<td class='main'><a href='/'>index</a> : <a title='aports' href='/aports/'>aports</a></td><td class='form'><form method='get'>
<select name='h' onchange='this.form.submit();'>
<option value='1.10-stable'>1.10-stable</option>
<option value='1.9'>1.9</option>
<option value='2.0-stable'>2.0-stable</option>
<option value='2.1-stable'>2.1-stable</option>
<option value='2.2-stable'>2.2-stable</option>
<option value='2.3-stable'>2.3-stable</option>
<option value='2.4-stable'>2.4-stable</option>
<option value='2.5-stable'>2.5-stable</option>
<option value='2.6-stable'>2.6-stable</option>
<option value='2.7-stable'>2.7-stable</option>
<option value='3.0-stable'>3.0-stable</option>
<option value='3.1-stable'>3.1-stable</option>
<option value='3.10-stable'>3.10-stable</option>
<option value='3.11-stable'>3.11-stable</option>
<option value='3.12-stable'>3.12-stable</option>
<option value='3.13-stable'>3.13-stable</option>
<option value='3.14-stable'>3.14-stable</option>
<option value='3.15-stable'>3.15-stable</option>
<option value='3.16-stable'>3.16-stable</option>
<option value='3.17-stable'>3.17-stable</option>
<option value='3.2-stable'>3.2-stable</option>
<option value='3.3-stable'>3.3-stable</option>
<option value='3.4-stable'>3.4-stable</option>
<option value='3.5-stable'>3.5-stable</option>
<option value='3.6-stable'>3.6-stable</option>
<option value='3.7-stable'>3.7-stable</option>
<option value='3.8-stable'>3.8-stable</option>
<option value='3.9-stable'>3.9-stable</option>
<option value='master' selected='selected'>master</option>
<option value='patches/3538'>patches/3538</option>
<option value='patches/3539'>patches/3539</option>
<option value='patches/3808'>patches/3808</option>
<option value='patches/3888'>patches/3888</option>
<option value='patches/3913'>patches/3913</option>
<option value='patches/3977'>patches/3977</option>
<option value='patches/3998'>patches/3998</option>
</select> <input type='submit' value='switch'/></form></td></tr>
<tr><td class='sub'>Alpine packages build scripts
</td><td class='sub right'>uwsgi</td></tr></table>
<table class='tabs'><tr><td>
<a href='/aports/about/'>about</a><a href='/aports/'>summary</a><a href='/aports/refs/'>refs</a><a href='/aports/log/testing/amule/wxwidgets-3.2.patch'>log</a><a class='active' href='/aports/tree/testing/amule/wxwidgets-3.2.patch'>tree</a><a href='/aports/commit/testing/amule/wxwidgets-3.2.patch'>commit</a><a href='/aports/diff/testing/amule/wxwidgets-3.2.patch'>diff</a><a href='/aports/stats/testing/amule/wxwidgets-3.2.patch'>stats</a></td><td class='form'><form class='right' method='get' action='/aports/log/testing/amule/wxwidgets-3.2.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='search' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/aports/tree/'>root</a>/<a href='/aports/tree/testing'>testing</a>/<a href='/aports/tree/testing/amule'>amule</a>/<a href='/aports/tree/testing/amule/wxwidgets-3.2.patch'>wxwidgets-3.2.patch</a></div><div class='content'>blob: 04f5c92296e0b4d0e79c31735663c4f0136b1686 (<a href='/aports/plain/testing/amule/wxwidgets-3.2.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
<a id='n32' href='#n32'>32</a>
<a id='n33' href='#n33'>33</a>
<a id='n34' href='#n34'>34</a>
<a id='n35' href='#n35'>35</a>
<a id='n36' href='#n36'>36</a>
<a id='n37' href='#n37'>37</a>
<a id='n38' href='#n38'>38</a>
<a id='n39' href='#n39'>39</a>
<a id='n40' href='#n40'>40</a>
<a id='n41' href='#n41'>41</a>
<a id='n42' href='#n42'>42</a>
<a id='n43' href='#n43'>43</a>
<a id='n44' href='#n44'>44</a>
<a id='n45' href='#n45'>45</a>
<a id='n46' href='#n46'>46</a>
<a id='n47' href='#n47'>47</a>
<a id='n48' href='#n48'>48</a>
<a id='n49' href='#n49'>49</a>
<a id='n50' href='#n50'>50</a>
<a id='n51' href='#n51'>51</a>
<a id='n52' href='#n52'>52</a>
<a id='n53' href='#n53'>53</a>
<a id='n54' href='#n54'>54</a>
<a id='n55' href='#n55'>55</a>
<a id='n56' href='#n56'>56</a>
<a id='n57' href='#n57'>57</a>
<a id='n58' href='#n58'>58</a>
<a id='n59' href='#n59'>59</a>
<a id='n60' href='#n60'>60</a>
<a id='n61' href='#n61'>61</a>
</pre></td>
<td class='lines'><pre><code>diff --git a/src/muuli_wdr.cpp b/src/muuli_wdr.cpp
index 44f0eb6..64dbade 100644
<span style="color:#000000; font-weight:bold">--- a/src/muuli_wdr.cpp</span>
<span style="color:#0057ae">+++ b/src/muuli_wdr.cpp</span>
&#64;&#64; -181,10 +181,10 &#64;&#64; wxSizer *searchDlg( wxWindow *parent, bool call_fit, bool set_sizer )
     wxBoxSizer *item3 = new wxBoxSizer( wxHORIZONTAL );
 
     wxStaticText *item4 = new wxStaticText( parent, -1, _(&quot;Name:&quot;), wxDefaultPosition, wxDefaultSize, 0 );
<span style="color:#000000; font-weight:bold">-    item3-&gt;Add( item4, 0, wxALIGN_CENTER|wxALL, 5 );</span>
<span style="color:#0057ae">+    item3-&gt;Add( item4, 0, wxALIGN_CENTER_VERTICAL|wxALL, 5 );</span>
 
     CMuleTextCtrl *item5 = new CMuleTextCtrl( parent, IDC_SEARCHNAME, wxT(&quot;&quot;), wxDefaultPosition, wxSize(80,-1), wxTE_PROCESS_ENTER );
<span style="color:#000000; font-weight:bold">-    item3-&gt;Add( item5, 1, wxALIGN_CENTER|wxALL, 5 );</span>
<span style="color:#0057ae">+    item3-&gt;Add( item5, 1, wxALIGN_CENTER_VERTICAL|wxALL, 5 );</span>
 
     wxFlexGridSizer *item6 = new wxFlexGridSizer( 1, 0, 0, 0 );
 
&#64;&#64; -212,7 +212,7 &#64;&#64; wxSizer *searchDlg( wxWindow *parent, bool call_fit, bool set_sizer )
     wxCheckBox *item12 = new wxCheckBox( parent, IDC_FILTERCHECK, _(&quot;Filtering&quot;), wxDefaultPosition, wxDefaultSize, 0 );
     item6-&gt;Add( item12, 0, wxALIGN_CENTER|wxALL, 5 );
 
<span style="color:#000000; font-weight:bold">-    item3-&gt;Add( item6, 0, wxALIGN_CENTER, 0 );</span>
<span style="color:#0057ae">+    item3-&gt;Add( item6, 0, wxALIGN_CENTER_VERTICAL, 0 );</span>
 
     item1-&gt;Add( item3, 0, wxGROW|wxALIGN_CENTER_VERTICAL, 5 );
 
&#64;&#64; -925,17 +925,17 &#64;&#64; wxSizer *downloadDlg( wxWindow *parent, bool call_fit, bool set_sizer )
     item0-&gt;Add( item2, 0, wxALIGN_CENTER|wxLEFT|wxRIGHT, 5 );
 
     wxGauge *item3 = new wxGauge( parent, ID_HTTPDOWNLOADPROGRESS, 100, wxDefaultPosition, wxSize(-1,10), wxGA_SMOOTH|wxGA_PROGRESSBAR );
<span style="color:#000000; font-weight:bold">-    item0-&gt;Add( item3, 0, wxGROW|wxALIGN_CENTER_VERTICAL|wxALL, 5 );</span>
<span style="color:#0057ae">+    item0-&gt;Add( item3, 0, wxGROW|wxALIGN_CENTER_HORIZONTAL|wxALL, 5 );</span>
 
     wxStaticText *item4 = new wxStaticText( parent, IDC_DOWNLOADSIZE, _(&quot;Unknown size&quot;), wxDefaultPosition, wxDefaultSize, wxALIGN_CENTRE );
<span style="color:#000000; font-weight:bold">-    item0-&gt;Add( item4, 0, wxGROW|wxALIGN_CENTER_VERTICAL|wxALL, 5 );</span>
<span style="color:#0057ae">+    item0-&gt;Add( item4, 0, wxGROW|wxALIGN_CENTER_HORIZONTAL|wxALL, 5 );</span>
 
     wxStaticLine *item5 = new wxStaticLine( parent, -1, wxDefaultPosition, wxSize(20,-1), wxLI_HORIZONTAL );
<span style="color:#000000; font-weight:bold">-    item0-&gt;Add( item5, 0, wxGROW|wxALIGN_CENTER_VERTICAL|wxALL, 5 );</span>
<span style="color:#0057ae">+    item0-&gt;Add( item5, 0, wxGROW|wxALIGN_CENTER_HORIZONTAL|wxALL, 5 );</span>
 
     wxButton *item6 = new wxButton( parent, ID_HTTPCANCEL, _(&quot;Cancel&quot;), wxDefaultPosition, wxDefaultSize, 0 );
     item6-&gt;SetDefault();
<span style="color:#000000; font-weight:bold">-    item0-&gt;Add( item6, 0, wxALIGN_CENTER|wxALL, 5 );</span>
<span style="color:#0057ae">+    item0-&gt;Add( item6, 0, wxALIGN_CENTER_HORIZONTAL|wxALL, 5 );</span>
 
     if (set_sizer)
     {
diff --git a/src/utils/wxCas/src/wxcasprefs.cpp b/src/utils/wxCas/src/wxcasprefs.cpp
index d87aad3..c36ac64 100644
<span style="color:#000000; font-weight:bold">--- a/src/utils/wxCas/src/wxcasprefs.cpp</span>
<span style="color:#0057ae">+++ b/src/utils/wxCas/src/wxcasprefs.cpp</span>
&#64;&#64; -88,7 +88,7 &#64;&#64; WxCasPrefs::WxCasPrefs ( wxWindow * parent ) : wxDialog ( parent, -1,
 	                                       WxCasCte::DEFAULT_REFRESH_RATE ) );
 	m_refreshStaticText =
 	    new wxStaticText ( this, -1, _( &quot;Refresh rate interval in seconds&quot; ), wxDefaultPosition,
<span style="color:#000000; font-weight:bold">-	                       wxDefaultSize, wxALIGN_CENTRE );</span>
<span style="color:#0057ae">+	                       wxDefaultSize, wxALIGN_CENTER );</span>
 	m_refreshSBoxSizer-&gt;Add ( m_refreshSpinButton, 0, wxALL | wxALIGN_CENTER, 5 );
 	m_refreshSBoxSizer-&gt;Add ( m_refreshStaticText, 1, wxALL | wxALIGN_CENTER, 5 );
 
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated by <a href='https://git.zx2c4.com/cgit/about/'>cgit v1.2.3</a> (<a href='https://git-scm.com/'>git 2.25.1</a>) at 2023-03-04 15:27:12 +0000</div>
</div> <!-- id=cgit -->
</body>
</html>
